src/components/Pagination/index.js

```diff
@@ -58,6 +58,10 @@ const Pagination = ({
 }) => {
   const pagesInRange = pageNumbersList(currentPage, totalPages);
 
+  if (totalPages <= 1) {
```

Comment:
Could probably move this to before `pageNumbersList` since you don't really need that. But that's just optimisation